client/platform/web-girder/views/RunTrainingMenu.vue

```diff
@@ -161,6 +184,18 @@ export default defineComponent({
               label="Configuration File"
               :items="trainingConfigurations.configs"
             />
+            <v-file-input
+              v-if="labelText"
+              v-model="labelFile"
+              clearable
+            />
+            <import-button
+              name="Upload labels.txt File"
+              icon="mdi-folder-open"
+              open-type="txt"
+              class="grow"
+              @open="openTxt"
+            />
```

Comment:
Two notes:

* Is the import button necessary, or could we just use the `v-file-input` here?  In other places in the app where file inputs are expected (Such as annotation file during import) only the `v-file-input` is used.
* I don't think it's clear to the user what this is or that it's optional.  Perhaps we need a short description and a link to documentation.  I don't think matt has written any documentation about this file, so we should probably add it somewhere in our docs.

Reply:
I have changed the UI to use `v-file-input` instead of the import button. This also removes the need to use the `OpenFromDisk` method. 
I added the optional verbiage but I still don't feel I understand the contents of labels.txt file enough to provide meaningful documentation on it.

Reply:
should be set now